example-application/libraries/message-queue-client.js

```diff
@@ -9,25 +10,30 @@ class MessageQueueClient extends EventEmitter {
   constructor(customMessageQueueProvider) {
     super();
     this.isReady = false;
+    this.requeue = true; // Tells whether to return failed messages to the queue
 
     // To facilitate testing, the client allows working with a fake MQ provider
     // It can get one in the constructor here or even change by environment variables
     if (customMessageQueueProvider) {
       this.messageQueueProvider = customMessageQueueProvider;
-    } else if (process.env.MESSAGE_QUEUE_PROVIDER === 'real') {
-      this.messageQueueProvider = amqplib;
-    } else {
+    } 
+    else if(process.env.USE_FAKE_MQ === 'true'){
       this.messageQueueProvider = new FakeMessageQueueProvider();
     }
+    else {
+      this.messageQueueProvider = new FakeMessageQueueProvider();
```

Comment:
Shouldn't this be `amqp`? In the API route definition, you don't pass a provider which means you use the fake one by default